### ISSUE TITLE
Issues when creating BrowserWindow with alwaysOnTop

### ISSUE THREAD

**Issue by Inori** [electron/electron#8287]
* Electron version:electron-v1.4.13-win32-ia32
* Operating system:Win7 X64

### Expected behavior

The window shows normally.

### Actual behavior

The Window not shown.

### How to reproduce
Code modified from electron-quick-start-master, only change the window options.
If I create a window with these options, the window will not show:
```
  mainWindow = new BrowserWindow({width: 800, 
    height: 600,
    frame: false,
    transparent: true,
    minimizable: false,
    maximizable: false,
    alwaysOnTop: true,
    skipTaskbar: true,
    focusable: false,
    })
```
If I comment the `alwaysOnTop` option, it appears as expected:
![qq 20161227111743](https://cloud.githubusercontent.com/assets/6732524/21489550/dedf4832-cc26-11e6-8438-1448f939b9c0.png)

Actruall I'm trying to use this library:  [electron-notifications](https://www.npmjs.com/package/electron-notifications) in my project. It just uses window options as I describe above.

**Comment by liusy182** [comment]
Interestingly if we delay setting `alwaysOnTop` things will work.

```
    var win = new BrowserWindow({
      width: 800, 
      height: 600,
      frame: false,
      transparent: true,
      minimizable: false,
      maximizable: false,
      focusable: false
    });

    setTimeout( () => win.setAlwaysOnTop(true), 1000);

```

**Comment by zhouxianjun** [comment]
win.once('show', () => setTimeout( () => win.setAlwaysOnTop(true), 1000));

**Comment by sofianguy** [comment]
Thanks for contributing!

We are no longer releasing bug fixes for versions of Electron older than 1.6x. We prefer to only keep issues that are on our immediate radar.

If you are still having the same issue in an up-to-date version, feel free to reach out and we'll reopen.
